src/client/src/app/basics/lazy-element.jsx

```diff
@@ -47,6 +47,6 @@ export default class LazyElement extends React.Component {
             });
         }
 
-        return element;
+        return <Suspense fallback='loading...'>{element}</Suspense>
```

Comment:
Can you change this to use a blank span or something? Otherwise when the components are loading it will say "loading..." all over the place.

Reply:
I opened #1244 to resolve this.